examples/js/controls/OrbitControls.js

```diff
@@ -961,13 +959,13 @@ THREE.OrbitControls = function ( object, domElement ) {
 
 	function onMouseUp( event ) {
 
```

Comment:
Prevents "stickiness" by removing "pointerup" and "pointermove" even if controls are disabled mid-gesture.